init.lua

```diff
@@ -50,7 +50,7 @@ minetest.register_entity("gauges:hp_bar", {
 			return
 		end
 
-		local hp = min(player:get_hp(), 20)
+		local hp = min(math.ceil((player:get_hp() / player:get_properties().hp_max) * 20), 20)
```

Comment:
Should we also handle the case where `hp_max` is greater than 20? This would be a good opportunity to do it at the same time.